src/shared/cache/GenomeNexusCache.ts

```diff
@@ -22,6 +23,20 @@ export function fetch(queries: Mutation[]): Promise<VariantAnnotation[]> {
     }
 }
 
+// fetch function for GRCh38
```

Comment:
is there a reason we duplicate this function below?

Reply:
these cache should be able to use the client that you've set up now in the store.  they should not need to know about which GRCh38 or 37.

Reply:
same reason as above